syft/workers/base.py

```diff
@@ -193,15 +193,15 @@ def send(self, tensor, workers, ptr_id=None):
             ptr_id = int(10e10 * random.random())
 
         # Send the object
-        self.send_obj((ptr_id, tensor), worker)
+        self.send_obj(tensor, worker)
 
         pointer = tensor.create_pointer(
-            owner=self, location=worker, id_at_location=ptr_id, register=True
+            owner=self, location=worker, id_at_location=tensor.id, register=True, ptr_id=ptr_id
```

Comment:
split these up so that one arg is on each line (I think autopep8 does this)

Reply:
Done!